packages/core/src/components/way-autosuggest/way-autosuggest.tsx

```diff
@@ -0,0 +1,265 @@
+import { Component, Method, Element, Event, EventEmitter, Listen, State, Prop, Watch, h } from '@stencil/core';
+import { IFormElementData } from '../../types/form';
+import { ISelectOption, ISelectConfig } from '../../types/select';
+
+@Component({
+  tag: 'way-autosuggest',
+  styleUrl: 'way-autosuggest.scss',
+  shadow: true,
+})
+export class W2wSelect {
+  private optionListEl?: HTMLInputElement;
+  private inputEl?: HTMLInputElement;
+  @Element() el!: HTMLWayAutosuggestElement;
+
+  @Prop() options?: Array<ISelectOption> = [];
+  @Prop() config?: ISelectConfig = {};
+  @Prop() validation?: (value: any) => string[];
+  @Prop() value?: string | Array<{ label: string; value: any }>;
+  @Prop() name: string;
+
+  @State() inputValue: string = '';
+  @State() hasFocus = false;
+  @State() filteredOptions: Array<ISelectOption> = this.options;
+  @State() localSelected = [];
+  @State() localConfig: ISelectConfig = {
+    selectedText: 'selected',
+    maxTagWidth: '10rem',
+    tagColor: 'green',
+    ...this.config,
+  };
+  @State() highlightIndex: number = 0;
+
+  // use until clicking outside works
+  @State() scrollPos: number = 0;
+
+  @Method()
+  async getValue() {
```

Comment:
I expect a `value` prop instead of a `getValue()` method.

Reply:
done